pkgs/test/stdenv/hooks.nix

```diff
@@ -0,0 +1,130 @@
+{ stdenv, pkgs }:
+
+# ordering should match defaultNativeBuildInputs
```

Comment:
I'd just embrace the chaos and/or sort alphabetically.
This is about the attrs below?

Reply:
i sorted defaultNativeBuildInputs alphabetically instead